Sources/UIKitExtensions/UIProgressView+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIProgressViewFucntions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+import UIKit
+public extension ConfigurationSet where Base: UIProgressView {
+    
+    @available(iOS 5.0, *)
+    func setProgress(_ progress: Float, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UIProgressView: Base ) in
+            UIProgressView.setProgress(progress, animated: animated)
```

Comment:
This function is only useful for existing UIProgressView's whose progress change should be animated.
The progress Property ist writable directly so there is no need for this function.